Mage.Tests/src/test/java/org/mage/test/cards/triggers/damage/ObNixilisCaptiveKingpinTest.java

```diff
@@ -0,0 +1,174 @@
+package org.mage.test.cards.triggers.damage;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import mage.counters.CounterType;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestCommander4Players;
+
+public class ObNixilisCaptiveKingpinTest extends CardTestCommander4Players {
+
+    @Test
```

Comment:
Looks like a details list from your comment about possible use cases can be useful here (just copy as comment).

![IMG_0022](https://github.com/magefree/mage/assets/8344157/1801931e-6e86-4d6e-8466-3a161ecfe87e)